settings.py

```diff
@@ -33,9 +33,13 @@
 
 HOMEPAGE = "http://chaosthebot.com"
 
-# TEST SETTING PLEASE IGNORE
+# TEST SETTING
```

Comment:
No. No. No. You're suppose to pay very close attention to that variable. It's `TSPI` that you should ignore...